penelope_aerospace_pl_msgs/msg/ModuleState.msg

```diff
@@ -16,5 +16,6 @@ uint8 ACCEPTED=1
 uint8 EXECUTING=2
 uint8 PAUSED=7
 uint8 CANCELING=3
+uint8 INITIALIZING=4
```

Comment:
This should go in the `DVMState`. The `ModuleState` is only sent in feedback messages and is shared across all modules. I have not seen any module send feedback while still initializing the action (e.g. verifying if the goal is valid and should be accepted.)

This message might actually be removed based on the usage of it.

Reply:
I updated the code in the penelope_TNO_module. Now it's using the EXECUTING. could you fix the ModuleState.msg?